Add links to user guide and pre-built exp matrix

xiaolongchen0627 opened the pull request.

xiaolongchen0627 committed:
add user guide

xiaolongchen0627 committed:
Update README.md

xiaolongchen0627 committed:
add link to pre-built exp matrix

xiaolongchen0627 committed:
Update README.md

xiaolongchen0627 committed:
Update README.md

zaeleus commented on src/core/README.md:
```diff
@@ -12,6 +12,7 @@
   * [MEME Suite] =4.9.0
   * [twoBitToFa]\*
   * [variants2matrix] (See below.)
+  * [pre-built expression matrix for cancers]
```
This is not a hard dependency. Precalculated expression matrices are discussed under the [References](https://github.com/stjude/cis-x/tree/fc8639c5bf1ab127583c18714f3150a6966bd945/src/core#references) section of the README.

xiaolongchen0627 replied:
Oh, that is my bad. we can leave it as it was.

xiaolongchen0627 replied:
should I just change the README.md back and re-push it ?

zaeleus replied:
Yes, you can edit and repush anything in your `add-a-user-guide-link` branch.

xiaolongchen0627 replied:
I change it back, should I pull another request?

zaeleus replied:
No, changes are updated in this PR: https://github.com/stjude/cis-x/pull/2/commits/5e42db2e3b334fba3d92c9fad59bc95977c3b988

zaeleus reviewed (changes requested):
Delete `.DS_Store`.

xiaolongchen0627 committed:
cancel updates

xiaolongchen0627 committed:
request change from Yu

xiaolongchen0627 committed:
add a references part

Yu think we should add a reference to the docker part, making is easier to run docker version

xiaolongchen0627 committed:
Update README.md

xiaolongchen0627 committed:
Update README.md

zaeleus committed:
Remove .DS_Store

zaeleus committed:
readme: Move user guide information to intro

zaeleus committed:
readme: Note how seeds are generated in the Docker instructions

zaeleus committed:
readme: Fix Docker header level
